src/liballoc/heap.rs

```diff
@@ -16,6 +16,18 @@
 
 use core::{isize, usize};
 
+#[allow(improper_ctypes)]
+extern {
+    #[allocator]
```

Comment:
It looks like there's nothing gating the use of this attribute (I don't see `feature` added to this crate). Is there anything preventing this from being written on stable?